builder/googlecompute/tunnel_driver.go

```diff
@@ -0,0 +1,80 @@
+// +build !windows
+
+package googlecompute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+)
+
+func NewTunnelDriver() TunnelDriver {
+	return &TunnelDriverLinux{}
+}
+
+type TunnelDriverLinux struct {
+	cmd *exec.Cmd
+}
+
+func (t *TunnelDriverLinux) StartTunnel(cancelCtx context.Context, tempScriptFileName string) error {
```

Comment:
The way we invoke Windows SSM stuff is going to be pretty different because it uses vendored SDK bindings instead of shelling out to a separately-installed cli tool. I don't know how useful it'll be to try to generalize between the IAP and SSM when the majority of StartTunnel here is just os.exec boilerplate which I believe isn't relevant to SSM. 

However we _should_ make sure that the structure and naming conventions of the tunnel code are similar. I'll take a look at how Wilken named and structured the SSM code and see if I can make the IAP code map to it. 

In terms of generalizing between the unix and Windows stuff specifically for IAM -- I'll investigate whether some of this can be deduplicated _after_ I get Windows working/tested and figure out what even needs to be done for it to get the process to terminate correctly on windows vs unix.